pkg/common/utils.go

```diff
@@ -223,15 +209,15 @@ func ZeroTimeInUnixNano(t time.Time) *int64 {
 	return &tInt
 }
 
-func WaitForCondition(eval func() bool, interval time.Duration, timeout time.Duration) error {
+func WaitForCondition(interval time.Duration, timeout time.Duration, eval func() bool) error {
```

Comment:
Just curious, why we change the argument order? I assume we can do similar cleanup for k8hime module. And this changes of order will make difference between core and k8shim.

Reply:
Since the argument order of the WaitFor function is different from that of the WaitForCondition function, I need to replace the WaitFor function with the WaitForCondition function. However, since the WaitFor function is more frequently used in the project than the WaitForCondition function, I have decided to change the argument order of the WaitForCondition function.

Reply:
that makes sense to me